requirements.txt

```
@@ -1,6 +1,6 @@
 django>=2.2,<3.2
 swapper>=1.1.0,<1.2.0
-djangorestframework<3.12,>=3.11
+djangorestframework~=3.12
```

This is necessary otherwise it will be treated as `djangorestframework ==3.x`
```suggestion
djangorestframework~=3.12.0
```